src/host/main.cpp

```diff
@@ -79,10 +74,15 @@ int main(int argc, char** argv)
     return app.exit(e);
   }
 
+  LOG_INFO_FMT("Version: {}", ccf::ccf_version);
+
   std::string config_str = files::slurp_string(config_file_path);
   CCHostConfig config = nlohmann::json::parse(config_str);
   LOG_INFO_FMT("Configuration: {}", config_str);
 
+  LOG_FAIL_FMT(
```

Comment:
That seems oddly specific and yet context-free in the log?

Reply:
This is now gone.